src/System.Runtime.Numerics/src/System/Numerics/Complex.cs

```diff
@@ -271,13 +278,40 @@ public static Complex Acos(Complex value)
 
         public static Complex Tan(Complex value)
         {
-            return (Sin(value) / Cos(value));
+            // tan z = sin z / cos z, but to avoid unnecessary repeated trig computations, use
+            //   tan z = \frac{\sin(2x) + i \sinh(2y)}{\cos(2x) + \cosh(2y)}
```

Comment:
This is just by TeX background showing though. Agreed, for formulas this simple, it's unnecessary. Will change in PR update.